src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -190,7 +190,7 @@ const DriverOverviewCard: React.FC<DriverOverviewCardProps> = ({ data }) => {
                 <View style={[styles.infoAndLogoContainer, styles.flexRow]}>
                     <View style={styles.informationContainer}>
                         <Text style={styles.h1text}>Driver Overview</Text>
-                        <Text style={styles.h2text}>Driver Name: {data.driverName}</Text>
+                        <Text style={styles.h2text}>Driver Name: {data.driverName ?? ""}</Text>
```

Comment:
Do we want this to say "Unknown driver" here like in the other place? May be worth pulling out into a constant somewhere

Reply:
Sure